config/toggles.js

```diff
@@ -116,6 +118,9 @@ const longTermToggles = {
   enableSampleStoreInfoLogging: environmentVariableTrue(pe,
     'ENABLE_SAMPLE_STORE_INFO_LOGGING'),
 
+  // Enable graceful shutdown handling event
+  enableSigtermEvent: environmentVariableTrue(pe, 'ENABLE_SIGTERM_EVENT'),
```

Comment:
This should be a short-term toggle - this is functionality we intend to make permanent, the use of the toggle is just to ensure smooth rollout

Reply:
Done